app/jobs/my_facilities_download_job.rb

```diff
@@ -0,0 +1,8 @@
+class MyFacilitiesDownloadJob < ApplicationJob
```

Comment:
Can you match the existing sidekiq jobs for this?

So just a normal class (no ApplicationJob)  `  include Sidekiq::Worker`